tmva/pymva/test/generateKerasModelFunctional.py

```diff
@@ -0,0 +1,24 @@
+import os
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
+
+import keras
```

Comment:
This works for old Keras version but not for tensorflow.keras. 
I think we should support first tensorflow keras and eventually old versions. 

For changing to tf.keras see for example the tutorials 
https://github.com/root-project/root/tree/master/tutorials/tmva/keras

Reply:
Modified code for importing from tensorflow.keras